stdlib/ballerina-builtin/src/main/ballerina/ballerina/io/natives.bal

```diff
@@ -18,28 +18,28 @@ public struct TextRecordChannel{
 @Return {value:"CharacterChannel converted from ByteChannel"}
 public native function <ByteChannel channel> toCharacterChannel(string encoding)(CharacterChannel);
 
-@Description {value:"Function to convert a CharacterChannel to TextRecordChannel"}
+@Description {value:"Function to convert a CharacterChannel to DelimitedRecordChannel"}
 @Param {value:"channel: The CharacterChannel to be converted"}
 @Param {value:"recordSeparator: Terminating expression to distinguish between records"}
 @Param {value:"fieldSeparator: Terminating expression to distinguish between fields"}
-@Return {value:"TextRecordChannel converted from CharacterChannel"}
+@Return {value:"DelimitedRecordChannel converted from CharacterChannel"}
 public native function <CharacterChannel channel> toTextRecordChannel(string recordSeparator,
```

Comment:
Shall we rename this method as well?

Reply:
@wggihan let's track the renaming of these methods separately. As this doesn't go inline with the jira.